app/views/static_pages/contact.html.erb

```diff
@@ -1,4 +1,4 @@
-<% provide(:title, "Contact") %>
+<% provide :title, "Contact" %>
 <h1>Contact</h1>
 <p>
   Contact the Ruby on Rails Tutorial about the sample app at the
```

Comment:
Tìm hiểu về I18n là gì em nhé?
Sau đó sử dụng vào toàn bộ cho các file html nhé
chị comment 1 file này thôi nhé, các file html khác em sẽ fix tương tự nhé